lib/email_auth.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+#email_auth.py
+#
+#Combine standard login with option of OAuth2.0
+#
```

Comment:
nit: import order should be consistent. To be honest I don't know what the proper flow is but I generally do all "import" statements first, then all "from x import y" statements

Reply:
Fixed imports